### ISSUE TITLE
Deploy workflow using a definition string

### ISSUE THREAD

**Issue by pedesen** [creditsenseau/zeebe-client-node-js#86]
It would be nice, if I could also deploy a workflow using a UTF8-encoded string in addition to providing a file path. This would make sense in context of the [Node-RED zeebe nodes](https://github.com/pedesen/node-red-contrib-zeebe), where the source of a workflow definition is more diverse.

A solution might be to optionally bypass readFile somehow here: https://github.com/creditsenseau/zeebe-client-node-js/blob/master/src/zb/ZBClient.ts#L184

**Comment by jwulf** [comment]
Yes, it could be extended to take an object, allowing different things to be passed in.
